rest/index.py

```diff
@@ -22,62 +24,83 @@
 from bson import json_util
 from bson.codec_options import CodecOptions
 
-from mozdef_util.elasticsearch_client import ElasticsearchClient, ElasticsearchInvalidIndex
+from mozdef_util.elasticsearch_client import (
+    ElasticsearchClient,
+    ElasticsearchInvalidIndex,
+)
 from mozdef_util.query_models import SearchQuery, TermMatch
 
 from mozdef_util.utilities.logger import logger, initLogger
 from mozdef_util.utilities.toUTC import toUTC
 
 
 options = None
-pluginList = list()   # tuple of module,registration dict,priority
+pluginList = list()  # tuple of module,registration dict,priority
+
+# The name of the MongoDB database that stores duplicate chains.
+DUP_CHAIN_DB = "duplicatechains"
+
+# The format string that UTC-based timestamps are encoded as.
+DUP_CHAIN_DATE_FMT = "%Y/%m/%d %H:%M:%S"
+
+
+class StatusCode(enum.IntEnum):
+    """A simple enumeration of common status codes.
+    """
+
+    OK = 200
+    BAD_REQUEST = 400
+    INTERNAL_ERROR = 500
 
 
 def enable_cors(fn):
-    ''' cors decorator for rest/ajax'''
+    """ cors decorator for rest/ajax"""
```

Comment:
In the spirit of minimizing changes, can we revert the change from single quotes to double quotes as well as the line length fixes? There's a lot of additional changes in this file that aren't specifically related to the work done in the PR, so reviewing this is quite a pain currently.

Reply:
I'm somewhat inclined to suggest we leave these changes in.  The bulk of these changes resulted from running the [black](https://github.com/psf/black) code formatter on this and several other files.  This tremendously helped to eliminate flake errors as well as to make code consistent.  I'd like to argue this is a good step towards making code style & formatting consistent, and I don't think it would be a good use of time to manually comb through the code to re-introduce non-conformant formatting.  The links I've provided in my opening comment should direct you to the portions of code that are critical to the requested review.

Do let me know if you disagree.

Reply:
I disagree, I see the value with running the black code formatter on the codebase, but not as part of these specific changes, as it makes reviewing this PR significantly more difficult. These files were previously passing our flake8 tests, so there shouldn't have been previous errors (excluding the 5 flake8 tests we currently specifically exclude in https://github.com/mozilla/MozDef/blob/master/.flake8#L18-L22).

I think the work of :
  1. Reverting the file back to the state before this PR
  2. Manually copying/pasting the new code required for this PR
  3. Committing file
Would significantly help speed up the review process.